samples/geolocation-weather/src/helpers.ts

```diff
@@ -0,0 +1,31 @@
+const getClientAddressFromRequest = (req: Request): string | null => {
+    const clientAddress = req.headers.get("spin-client-addr");
```

Comment:
I am not sure if [this issue](https://github.com/spinframework/spin/issues/2989) would lead us to use a different approach here.